### PR TITLE
Detectors are renamed -  FT0 and FV0 

### PR DESCRIPTION
**Pull request by AllaMaevskaya**

### CODE REVIEW

File: Detectors/Base/include/DetectorsBase/DCAFitter.h [hunk 1]
```diff
@@ -227,7 +227,7 @@ class DCAFitter
     setUseAbsDCA(false); // by default use weighted DCA definition (much slower)
   }
 
-  ///< number of validated V0 candidates (at most 2 are possible)
+  ///< number of validated FV0 candidates (at most 2 are possible)
```
Comment (shahor02) [inline Detectors/Base/include/DetectorsBase/DCAFitter.h]: I've already commented on this in the previous PR https://github.com/AliceO2Group/AliceO2/pull/2255#discussion_r307315630, these are secondary vertices V0s, please revert this file

File: Detectors/Base/test/testDCAFitter.cxx [hunk 2]
```diff
@@ -44,7 +44,7 @@ BOOST_AUTO_TEST_CASE(PairDCAFitter)
 {
   double bz = 5.0;
 
-  // create V0
+  // create FV0
```
Comment (shahor02) [inline Detectors/Base/test/testDCAFitter.cxx]: this also

File: Detectors/FIT/FV0/simulation/src/Detector.cxx [hunk 3]
```diff
@@ -131,10 +131,10 @@ Bool_t Detector::ProcessHits(FairVolume* v)
   return kTRUE;
 }
 
-o2::v0::Hit* Detector::addHit(Int_t trackId, Int_t cellId,
-                              TVector3 startPos, TVector3 endPos,
-                              TVector3 startMom, double startE,
-                              double endTime, double eLoss, Int_t particlePdg)
+o2::fv0::Hit* Detector::addHit(Int_t trackId, Int_t cellId,
+                               TVector3 startPos, TVector3 endPos,
```
Comment (shahor02) [inline Detectors/FIT/FV0/simulation/src/Detector.cxx]: No need to use TObject based virtual object to transfer the position/momentum, please change this to ``Point3D<float>`` or ``Vector3D<float>``, also in related methods

File: Detectors/TPC/calibration/include/TPCCalibration/CalibPulser.h [hunk 4]
```diff
@@ -169,7 +169,7 @@ class CalibPulser : public CalibRawBase
     /// return pointer to T0 histogram
     TH2S* getHistoT0(ROC roc, Bool_t force)
     {
-      return getHistogram(roc, mT0Histograms, mNbinsT0, mXminT0, mXmaxT0, "T0", force);
+      return getHistogram(roc, mT0Histograms, mNbinsT0, mXminT0, mXmaxT0, "FT0", force);
```
Comment (shahor02) [inline Detectors/TPC/calibration/include/TPCCalibration/CalibPulser.h]: This one also was commented in the prev. PR https://stackoverflow.com/questions/32766151/check-for-optionally-targets-in-cmake-that-are-not-in-the-correct-order, it is for TPC start time, revert this file please

File: Detectors/TPC/calibration/src/CalibPulser.cxx [hunk 5]
```diff
@@ -220,7 +220,7 @@ void CalibPulser::analyse()
 void CalibPulser::dumpToFile(const std::string filename)
 {
   auto f = std::unique_ptr<TFile>(TFile::Open(filename.c_str(), "recreate"));
-  f->WriteObject(&mT0,    "T0");
+  f->WriteObject(&mT0, "T0");
```
Comment (shahor02) [inline Detectors/TPC/calibration/src/CalibPulser.cxx]: this one too.

File: EventVisualisation/DataConverter/include/EventVisualisationDataConverter/ConversionConstants.h [hunk 6]
```diff
@@ -20,24 +20,24 @@
 #include <string>
 
 enum ETrackType {
-    Standard,
+  Standard,
```
Comment (shahor02) [inline EventVisualisation/DataConverter/include/EventVisualisationDataConverter/ConversionConstants.h]: This one also is for secondary vertices, revert.

File: Detectors/TPC/calibration/src/CalibPulser.cxx (hunk 5 above)
Comment (AllaMaevskaya) [reply Detectors/TPC/calibration/src/CalibPulser.cxx]: sorry, I did not check, I was sure he fixed this...

File: Detectors/FIT/FV0/simulation/src/Detector.cxx (hunk 3 above)
Comment (AllaMaevskaya) [reply Detectors/FIT/FV0/simulation/src/Detector.cxx]: Fixed now

**Comment by AllaMaevskaya** [comment]
We satisfied all requests, yes?

**Comment by shahor02** [comment]
Hi @AllaMaevskaya 
almost, I've fixed the rest, will squash and merge once tests are passed.

**Pull request merged**